pyfarm/master/etc/master.yml

```diff
@@ -12,7 +12,207 @@
+# a local SQLite database is used.  For production access should be
+# configured to a more standard SQL database.  MySQL or Postgres are
+# the two primary databases PyFarm is regurarly tested against.  For
+# more information on the expected format of this variable see
+# SQLAlchemy's documentation:
+#   https://sqlalchemy.readthedocs.org/en/latest/core/engines.html#database-urls
+database: "sqlite:///pyfarm.sqlite"
+
+
+# The broker that PyFarm's scheduler should use.  For debugging and
+# development running Redis is the simplest.  For large deployments, or
+# to understand the format of this variable, see:
+#   http://celery.readthedocs.org/en/latest/configuration.html#broker-url
+scheduler_broker: "redis://"
+
+
+# The URL the master is running on.  This is used to form internal
+# urls and other information.
+base_url: http://127.0.0.1:5000/
+
+
+# The name of this render farm, normally this can remain the default value.  If
+# you're running multiple farms this allows you to only accept agents to your
+# master that match your current farm name.
+farm_name: ""
+
+
+# The secret key which is used by several components of Flask
+# for form validation, salting of secrets, etc.
+#
+# **Production Note**: This value should be random, consistent between
+# frontends, and kept secret.  Do not use the value below for
+# production.
+secret_key: pyfarm
+
+
+# The key used for signing the csrf token.
+#
+# **Production Note**: This value should be random, consistent between
+# frontends, and kept secret.  Do not use the value below for
+# production.
+csrf_session_key: pyfarm
+
+
+# The prefix of the URL from which the API will operate on.  This should
+# not generally be changed unless you are operating different versions
+# of the API at the same time from one web server.
+api_prefix: /api/v1
+
+
+# The URL template we use to communicate with the agent.
+agent_api_url_template: http://{host}:{port}/api/v1
+
+
+
+# Enables or disable the login functionality.  This can be used when
+# debugging or doing development but should not be changed for
+# production.
+login_disabled: false
+
+
+# The amount of time the 'remeber me' cookie should persist.  The keys
+# and values here are passed into a `timedelta` object as keywords.
+cookie_duration:
+  hours: 16
+
+
+# When true json output from te APIs will be reformatted to
+# be more human readable.
+pretty_json: false
+
+
+# When true all SQLAlchemy queries will be echoed.  This is useful
+# for debugging the SQL statements being run and to get an idea of
+# what the underlying ORM may be doing.
+echo_sql: false
+
+
+# When true the application will automatically create users in
+# PyFarm's database if they do not exist already.  Setting this
+# to false will cause an exception to be raised if the user in
+# question does not exist.
+autocreate_users: true
+
+
+# When autocreating users, use this email address as a template.  For example:
+#   {username}@example.com
```

Comment:
The given example will confuse the yaml parser, leading to errors like this:

    Failed to load '/etc/pyfarm/master/master.yml': while scanning for the next token
    found character '@' that cannot start any token